Client/src/main/java/com/bbd/BeanClient/UserInput.java

```diff
@@ -115,8 +120,41 @@ private static void post(List<String> commandElements){
     }
 
     private static void profile(List<String> commandElements){
+        String username = "";
+        String favBean = "";
+        String bio = "";
+
         //todo I am not gonna make this now LOL
         //the stuff to make a profile needs to be in the command or the user will be prompted to make one, still unsure
+        System.out.println("Please enter a username:");
+        username = scanner.nextLine();
+        //CHECK TO MAKE SURE USERNAME ISNT TAKEN
+        System.out.println("Please enter a short bio about yourself:");
+        bio = scanner.nextLine();
+        //TODO: show all of the beans here
+        System.out.println("Please enter your favorite bean from the available list: ");
+        System.out.println("**Showing all beans here!!**");
+        favBean = scanner.nextLine();
```

Comment:
Remember you have access to the commandElements list that was passed in. This might already contain some of the info that you try to capture here?

Reply:
I assume you're going to improve on this when other functionality is available, so I'm approving this for now